presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -153,6 +154,12 @@ public boolean canReplicate()
         }
     }
 
+    public enum PartialMergePushdownStrategy
+    {
+        NONE,
+        PUSHDOWN_BUT_NOT_THROUGH_JOIN_AND_AGGREGATION
```

Comment:
Any thoughts about the name (looks long but I don't know how to make it better...) ? @rschlussel